capstone-app/src/app/modules/maps/map/map.component.scss

```diff
@@ -0,0 +1,17 @@
+.mapContainer {
+  margin-left: 3%;
+  margin-right: 3%;
+}
+
+.search-bar {
+  margin-left: 3%;
+  padding-bottom: 0;
+  text-align: center;
+}
+
+
+.buttonDelete.mat-raised-button {
+ float: right;
```

Comment:
is there any way you can use flex instead of float for this

Reply:
+1

Reply:
flex doesn't move it like float does.